src/templates/coveringLetter/customTemplate.tsx

```diff
@@ -1,27 +1,29 @@
-import React, { FunctionComponent } from "react";
```

Comment:
please make this repo cleaner otherwise I will ask to move it to tt:
- readme to update
- purpose (what's the diff with the template)
- filenames
- ci
- documentation

Reply:
ok i updated the stuffs, only left CI which i have requested.